src/app/refs_page.rs

```diff
@@ -83,12 +78,12 @@ impl<'repo> Drawable<'repo> for RefsPage<'repo> {
     fn title(&self) -> String {
         if let Some(path) = self.repo.path().parent() {
             if let Some(name) = path.file_name() {
-                format!("{}", name.to_string_lossy())
+                name.to_string_lossy().to_string()
```

Comment:
I feel it is more explicit to use `to_string` to convert this into a string instead of format(..), but I'm not sure if there are some minor differences. However, I did not notice anything look out of place after this.

Reply:
Yes, I fixed a number of these; this was very early learning